packages/contracts-bedrock/scripts/upgrades/PostSherlockL2.s.sol

```diff
@@ -299,4 +299,13 @@ contract PostSherlockL2 is SafeBuilder {
         require(set.BaseFeeVault != address(0), "no proxies for this network");
         return set;
     }
+
+    /**
```

Comment:
Ideally we can figure out long term how to not hardcode addresses so these same scripts can work across networks, or we have it so that all networks can use the same impls

Reply:
Will add a gardening ticket assigned to myself to design doc multi-network deployment configs. I know there are a variety out there, just need to find something reliable + extensible.

Reply:
Created [CLI-3901](https://linear.app/optimism/issue/CLI-3901/featcontracts-design-multi-network-deployment-configs)